Makefile

```
@@ -813,6 +813,11 @@ out/doc/api/all.json: $(apidocs_json) tools/doc/alljson.js
 docopen: $(apidocs_html)
 	@$(PYTHON) -mwebbrowser file://$(PWD)/out/doc/api/all.html
 
+.PHONY: docserve
+docserve: $(apidocs_html)
+	@$(PYTHON) -mwebbrowser http://localhost:8000/all.html
+	@$(PYTHON) -m http.server -d $(PWD)/out/doc/api
```

Since we don't include a space after the `-m` on line 818 and line 814, maybe omit it here too?

```suggestion
	@$(PYTHON) -mhttp.server -d $(PWD)/out/doc/api
```

I believe this line is Python 3 specific, which is probably fine, but just in case there's some desire on the part of @nodejs/python and/or @nodejs/build-files to preserve Python 2 compatibility even though it's been EOL'ed, I'm mentioning it.

---

I prefer the space. Je